### PR TITLE
feat(tx): Show script data when unable to decode

### PR DESCRIPTION
**Pull request by giovanecosta**
https://github.com/HathorNetwork/hathor-explorer/issues/45

Outputs with unknown script now will look like this: 
image

### CODE REVIEW

File: src/components/TxData.js [hunk 1]
```diff
@@ -215,15 +215,20 @@ class TxData extends React.Component {
...
-          return renderNanoContractMatchValues(decoded);
+          return renderNanoContractMatchValues(output.decoded);
         default:
-          return 'Unable to decode';
+          let script = output.script;
+          try {
+            script = atob(output.script)
+          } catch {}
+
+          return `Unable to decode script: ${script.trim()}`;
```
Comment (msbrogli) [inline src/components/TxData.js:231]: I feel we can't just do this because of injection attack vectors.

**Review by msbrogli — CHANGES_REQUESTED** [review]

File: src/components/TxData.js (hunk 1 above)
Comment (giovanecosta) [reply src/components/TxData.js:231]: @msbrogli It's already safe by design. If i want to put some HTML dynamically here, i have to do it explicitly.

Comment (msbrogli) [reply src/components/TxData.js:231]: Ok. Great. Thanks!

**Review by msbrogli — CHANGES_REQUESTED** [review]
I'm good to approve but your commit messages are not following the Semantic Commits.

**Review by pedroferreira1 — APPROVED** [review]

**Review by msbrogli — APPROVED** [review]

**Pull request merged**